test/e2e/LeakyIntegrationTest.php

```diff
@@ -110,7 +110,7 @@ public function doesLeakTestItself(): void
     /** @test */
     public function doesLeakAnAutoloader(): void
     {
-        spl_autoload_register(static function (string $className): bool {
+        spl_autoload_register(function (string $className): bool {
```

Comment:
Ah, just realized that this is intentional: the `spl_autoload_register()` part is just about checking that a new closure is added to the autoloader stack.

Reply:
For that, there is `doesLeakAStaticAutoloader`, isn't it?

I think this was mistakely fixed with CS fixer ;-)

Reply:
Good workaround with `$this` usage, thanks :-)